shaders/position.yaml

```diff
@@ -128,10 +129,11 @@ layers:
         draw:
             polygons:
                 style: buildings
-                order: 4
+                order: function() { return feature.sort_key; }
                 color: [.7,.7,.7]
                 extrude: true
             outline:
+                order: function() { return feature.sort_key + 1; }
```

Comment:
All of the examples here are outlines on top of polygons styles, so unless I'm misreading this there's no 'inline', just polygon fills. Or do polygon outlines work differently than I'm remembering?